Source/WebCore/platform/ThreadTimers.cpp

```diff
@@ -41,7 +41,7 @@
 
 namespace WebCore {
 
-WTF_MAKE_TZONE_ALLOCATED_IMPL(ThreadTimers);
+WTF_MAKE_COMPACT_TZONE_ALLOCATED_IMPL(ThreadTimers);
```

Comment:
Should we need `WTF_MAKE_COMPACT_TZONE_ALLOCATED(ThreadTimers)` as well?

Reply:
Ah, yeah, I think we don't.